codesystems/CodeSystem-England-MedicationRequestCategory.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CodeSystem xmlns="http://hl7.org/fhir">
+    <id value="England-MedicationRequestCategory"/>
+    <url value="https://fhir.nhs.uk/CodeSystem/England-MedicationRequestCategory"/>
+    <identifier>
+        <system value="https://fhir.nhs.uk/identifier/CodeSystem"/>
+        <value value="medicationrequest-category"/>
+    </identifier>
+    <version value="1.0.0"/>
+    <name value="EnglandMedicationRequestCategory"/>
+    <title value="England MedicationRequestCategory"/>
```

Comment:
MedicationRequestCategory -> Medication Request Category

Reply:
Done with the required changes.